### ISSUE TITLE
implement loss function for data imbalance

### ISSUE THREAD

**Issue by noiji** [noiji/sentencEmoji#2]

**Comment by noiji** [comment]
added weighted cross entropy loss function.
